.github/workflows/deploy.yml

```diff
@@ -29,6 +30,10 @@ jobs:
           cat build/data.json
           mustache build/data.json index.mustache > build/index.html
           cat build/index.html
+          
+      - name: Gather commits by day and file extension statistics
+        run: |
+          ./01stats.sh gecko-dev build
```

Comment:
What do we get out of this build step? Also I see that it takes pretty long time to complete.

Reply:
The goal is to move stats collection commands out of GitHub Actions YAML, so that they could be run standalone.

As I continued experiments in my `main` branch after opening this PR, more things started to creep in. The command that takes the most time is `git fetch --unshallow` added in the last commit to start playing with historical data.